<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Table</class>
 <widget class="QDialog" name="F_Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фильтр столов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <property name="windowFilePath">
   <string>z</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>408</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="minBet">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: transparent;
    border: none;
	padding: 5px;
	text-align: left;
}

QPushButton:hover {
font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>- Минимальная ставка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="categoryMinBet" native="true">
         <property name="styleSheet">
          <string notr="true">#categoryMinBet
{
    border: 2px solid black; 
    border-radius: 10px;
	background-color: rgb(198, 198, 198);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="allMinBet">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Нет лимита</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="start0end100">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>от 0 до 100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="start100end200">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>от 100 до 200</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="start200end300">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>от 200 до 300</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="yourLimitsMinBet">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>от</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startMinBet">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>до</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="endMinBet">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maxPlayers">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: transparent;
    border: none;
	padding: 5px;
	text-align: left;
}

QPushButton:hover {
font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>- Максимум игроков</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="categoryMaxPlayers" native="true">
         <property name="styleSheet">
          <string notr="true">#categoryMaxPlayers
{
    border: 2px solid black; 
    border-radius: 10px;
	background-color: rgb(198, 198, 198);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="allMaxPlayers">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Не важно</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="specQuantityMaxPlayrs">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="quantityMaxPlayrs">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="yourLimitsMaxPlayers">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>от</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startMaxPlayers">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>до</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="endMaxPlayers">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nameGame">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: transparent;
    border: none;
	padding: 5px;
	text-align: left;
}

QPushButton:hover {
font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>- Название игры</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="categoryNameGame" native="true">
         <property name="styleSheet">
          <string notr="true">#categoryNameGame
{
    border: 2px solid black; 
    border-radius: 10px;
	background-color: rgb(198, 198, 198);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="allNameGame">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Все</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="nameBlackJack">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>BlackJack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="nameDevytka">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Девятка</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reset">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid black;
    border-radius: 10px;
    padding: 5px;
    color: black;
}

QPushButton:hover {
    background-color: rgb(100, 88, 255);
	color: white;
}

QPushButton:pressed {
	background-color: rgba(220, 60, 190, 255);
}</string>
       </property>
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyFilter">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid black;
    border-radius: 10px;
    padding: 5px;
    color: black;
}

QPushButton:hover {
    background-color: rgb(100, 88, 255);
	color: white;
}

QPushButton:pressed {
	background-color: rgba(220, 60, 190, 255);
}</string>
       </property>
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
